src/main/scala/com/schuwalow/todo/Layers.scala

```diff
@@ -0,0 +1,41 @@
+package com.schuwalow.todo
+
+import zio.ZLayer
+import zio.blocking.Blocking
+import zio.logging.Logging.Logging
+import zio.logging.slf4j.Slf4jLogger
+
+import com.schuwalow.todo.config.AppConfigProvider
+import com.schuwalow.todo.config.ConfigProvider
+import com.schuwalow.todo.config.DbConfigProvider
+import com.schuwalow.todo.repository.DoobieTodoRepository
+import com.schuwalow.todo.repository.TodoRepository
+
+object Layers {
+
+  type Layer0Env =
+    ConfigProvider with Logging with Blocking
+
+  type Layer1Env =
+    Layer0Env with AppConfigProvider with DbConfigProvider
+
+  type Layer2Env =
+    Layer1Env with TodoRepository
+
+  type AppEnv = Layer2Env
+
+  object live {
+
+    val layer0: ZLayer[Blocking, Throwable, Layer0Env] =
+      Blocking.any ++ ConfigProvider.live ++ Slf4jLogger.make((_, msg) => msg)
+
+    val layer1: ZLayer[Layer0Env, Throwable, Layer1Env] =
+      AppConfigProvider.fromConfig ++ DbConfigProvider.fromConfig ++ ZLayer.identity
```

Comment:
Hey, awesome work! I Just have a question: what is the purpose of `ZLayer.identity`?

Reply:
Thank you :-). The `identity` just passes on the previous layer unmodified (remember that we have `type LayerN = PreviousLayer ++ Modules`).

Reply:
@mlangc Awesome, makes sense :)